src/index.tsx

```diff
@@ -19,3 +19,17 @@ ReactDOM.render(
   </Provider>,
   document.getElementById('app'),
 );
+
+// Register a service worker
+
+if ('serviceWorker' in navigator) {
+  (window || {} as any).addEventListener('load', function () {
+    navigator.serviceWorker.register('/sw.js').then(function (registration) {
```

Comment:
This is the format proposal, but for me, we should put enter before then. This code will look better.